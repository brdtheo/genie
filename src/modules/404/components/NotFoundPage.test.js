/**
 * @jest-environment jsdom
 */
import * as React from "react";
import renderer from "react-test-renderer";

import NotFoundPage from "./NotFoundPage";

describe("NotFoundPage component", () => {
  it("should render without throwing an error", () => {
    const tree = renderer.create(<NotFoundPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
