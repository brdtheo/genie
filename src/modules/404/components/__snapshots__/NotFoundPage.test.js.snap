// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component should render without throwing an error 1`] = `
<div
  className="w-full px-4 lg:px-0 lg:w-[56rem] m-auto "
  data-testid="container"
>
  <section
    className="pt-20 md:pt-28 pb-8 sm:pb-16 w-full"
  >
    <h2
      className="text-white text-xl font-semibold leading-none"
    >
      Not found
    </h2>
    <p
      className="text-white text-sm mt-4"
      data-testid="not-found-message"
    >
      The page you requested does not exist
    </p>
  </section>
</div>
`;
